### DISCUSSION TITLE
Add support for the -f flag for linting styles

### DISCUSSION THREAD

**Discussion by Veldrovive** [vercel/next.js#26387] [Ideas]
### Describe the feature you'd like to request

We use the `-f` flag to support a testing suite that uses the checkstyle format. Standard eslint supports this through the `-f checkstyle` flag which the current `next lint` command does not support.

### Describe the solution you'd like

Add support for the `-f` flag in the javascript api for eslint.
This [PR](https://github.com/vercel/next.js/pull/26280) adds a similar feature.

### Describe alternatives you've considered

You can also run `eslint` directly, but it is unfortunate to have to use a different command for testing in the CI than in dev as there may be unforeseen differences.

**Comment by housseindjirdeh** [comment]
PR to land this here:  https://github.com/vercel/next.js/pull/27052
